src-tauri/src/main.rs

```diff
@@ -141,6 +141,8 @@ fn main() {
             northstar::profile::delete_profile,
             northstar::profile::fetch_profiles,
             northstar::profile::validate_profile,
+            northstar::profile::delete_profile,
```

Comment:
Why did you put that in the tauri handler, since profiles can already be deleted?

Reply:
Was a leftover from a bad merge, removed with 275acb7e04a34f7d874cd13506537a780b36188e o7